dotcom-rendering/src/model/enhance-ad-placeholders.ts

```diff
@@ -5,20 +5,23 @@ import type { RenderingTarget } from '../types/renderingTarget';
 /**
  * Positioning rules:
  *
- * - Is further than 3 paragraphs in
- * - Is every 6 paragraphs
+ * - Is further than 3 blocks in
```

Comment:
Using `elements` instead of `blocks` would be more consistent with the logic used in liveblog pages. In liveblogs, an `element` is used for some content of the type `model.dotcomrendering.pageElements.???` whereas `blocks` is used to define a group of `elements` that make up a post

Reply:
I can switch to using elements 👍